azerothcore/azerothcore-wotlk
# fix(DB/npc_vendor): Empty vendors honor/arena S6 horde side

CHANGES PROPOSED:
Insert item list in npc_vendor Season 6 honor/arena horde side, now is empty.

ISSUES ADDRESSED:
#1944

TESTS PERFORMED:
tested successfully in-game

HOW TO TEST THE CHANGES:
change: Arena.ArenaSeason.ID = 6
restar server and check npc horde in halls of leyends - Orgrimar

OR check npc_vendor in world db table
-Sergeant Thunderhorn id 34036 vendor items list in season 6
-Blood Guard Zar'shi id 34061 vendor items list in season 6
-Doris Volanthius id 34058 vendor items list in season 6

Target branch(es):
Master
